src/99-appendices/03-hed.md

```diff
@@ -3,207 +3,179 @@
-               Action/Button press"
-        }
-   }
-}
-```
-
-## Annotating events by value type
-
-Each column of `*_events.tsv` containing non-categorical values usually represents a
-particular type of data, for example the `speed` of a stimulus object across the
-screen or the filename of the stimulus image.
-These variables could be annotated in the HED column of `*_events.tsv`.
-However, that approach requires repeating the values appearing in the individual
-columns in the HED column.
-A better approach is to annotate the type of value contained in each of these
-columns in the `*_events.json` sidecar.
-Value variables are annotated in a manner similar to categorical values,
-except that the HED string must contain exactly one `#` specifying a placeholder
-for the actual column values.
-Tools are responsible for substituting the actual column values for the `#` during analysis.
-
-Example: An `*_events.tsv` containing a categorical column (`trial_type`) and two value
-columns (`response_time` and `stim_file`).
+Example: An excerpt from an `events.tsv` file containing three columns
+(`trial_type`, `response_time`, and `stim_file`) in addition to
+the required `onset` and `duration` columns.
 
 ```Text
 onset  duration  trial_type  response_time stim_file
 1.2    0.6       go          1.435         images/red_square.jpg
 5.6    0.6       stop        1.739         images/blue_square.jpg
 ```
 
-Example: An accompanying `*_events.json` sidecar describing both categorical and value columns.
+The `trial_type` column in the above example contains a limited number of distinct
+values (`go` and `stop`).
+This type of column is referred to as a *categorical* column,
+and the column's meaning can be annotated by assigning HED tags to describe
+each of these distinct values.
+The JSON sidecar provides a dictionary of annotations for these categorical values.
+
+In contrast, the `response_time` and `stim_file` columns could potentially contain
+distinct values in every row.
+These columns are referred to as *value* columns and are annotated by creating
+a HED tag string to describe a general pattern for these values.
+The HED annotation for a value column must include a `#` placeholder,
+which tools will replace by the actual column value when the annotations
+are assembled for analysis.
+
+Example: An accompanying `_events.json` sidecar describing both categorical and
+value columns of the previous example.
+The `Duration` column is also annotated as a value column.
 
 ```JSON
 {
-   "trial_type": {
-       "LongName": "Event category",
-       "Description": "Indicator of type of action that is expected",
-       "Levels": {
-          "go": "A red square is displayed to indicate starting",
-          "stop": "A blue square is displayed to indicate stopping",
-       },
-       "HED": {
-          "go": "Event/Category/Experimental stimulus, Event/Label/RedSquare,
```

Comment:
The `eeg_hed_small` in the `bids_examples` repository on GitHub uses the preferred short form.

There are no examples currently in `bids_examples` that use long form, and I hesitate to put one there.

Instead, I just updated the PR with  a link to the `hed-examples` repository on `hed-standard` on GitHub.

In this repository we are developing various versions of repositories for testing, and I will put an `eeg_ds003654s_longform` and an `eeg_ds003654s_longform_defsexp` version on the repository. (The later refers to the option of expanding the definitions.)  

I will also add a `README` to the `datasets` directory there.

Reply:
On thinking about it.... we should probably also put a hybrid example to `bids-examples` so that unit tests verify that these forms are handled by the `bids-validator`.

Reply:
Great, this resolves these questions!